myuw/dao/term.py

```diff
@@ -318,3 +322,35 @@ def get_eod_specific_quarter_last_instruction(year, quarter):
     Only the summer full term is relevant.
     """
     return get_specific_term(year, quarter).get_eod_last_instruction()
+
+
+# The affilliation method caches values on the request object, but this one
+# is just designed to get values into our caching system.
+def _get_term_method(year, quarter):
+    def generated(request):
+        get_specific_term(year, quarter)
+    return generated
+
+
+def current_terms_prefetch(request):
```

Comment:
I expected that to be needed, but we get the terms around the current term by year/quarter.  That's why it does this work: methods.append(_get_term_method(year, quarter))